scripts/0-gold.js

```diff
@@ -1,20 +1,20 @@
-const { SignType, TransactionType, executeTransaction, balanceOf } = require('@algorand-builder/algob');
+const { balanceOf } = require('@algo-builder/algob');
+const { types } = require("@algo-builder/web");
 
 // a helper function used to create fund transaction
 function mkParam(senderAccount, receiverAddr, amount, payFlags) {
   return {
-    type: TransactionType.TransferAlgo,
-    sign: SignType.SecretKey,
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
     fromAccount: senderAccount,
     toAccountAddr: receiverAddr,
     amountMicroAlgos: amount,
     payFlags: payFlags
   };
 };
 
-
 // This is an entry function in our script (a default, exported function)
-async function run (runtimeEnv, deployer) {
+async function run(runtime, deployer) {
```

Comment:
does this need to be changed?

Reply:
Not needed but in algo-builder repo, we are using the same variable name. So thought of applying same here.